tests/functional/test_install.py

```diff
@@ -7,6 +7,7 @@
 
 import pytest
 
+import pip
```

Comment:
Since we're importing this only for the version, I'd prefer we import just the version and use that.

```py
from pip import __version__ as pip_current_version
```